resources/seccomp/x86_64-unknown-linux-musl.json

```diff
@@ -70,6 +70,14 @@
                 "syscall": "rt_sigreturn",
                 "comment": "rt_sigreturn is needed in case a fault does occur, so that the signal handler can return. Otherwise we get stuck in a fault loop."
             },
+            {
+                "syscall": "sigaltstack",
+                "comment": "sigaltstack is used by Rust stdlib to remove alternative signal stack during thread teardown."
+            },
+            {
+                "syscall": "tkill",
```

Comment:
Can we tighten this with an argument check?

Reply:
yes, good callout!